lib/compiler/src/artifact_builders/artifact_builder.rs

```diff
@@ -119,17 +119,10 @@ impl ArtifactBuild {
             compile_info,
             data_initializers,
             cpu_features: cpu_features.as_u64(),
-            module_start: None,
         };
         Ok(Self { serializable })
     }
 
-    /// Specify the fixed virtual memory address for the compiled module
```

Comment:
You can't just remove the whole module_start stuf...

We might not need it anymore, but that needs to be determined with John.

Reply:
Yeah that's what I wanted to sync on.

Reply:
I just wanted to show that it fixes the issue in the CI.

Reply:
@fschutt good news, we actually can remove it!

Reply:
Okay, then I can leave it as is then.